src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift

```diff
@@ -15,11 +15,6 @@ struct DeltaOnboardingNewVersionFeaturesViewModel {
 		
 		self.featureVersion = "2.19"
 		
-		self.newVersionFeatures.append(
-			// Rapid PCR tests
-			NewVersionFeature(title: AppStrings.NewVersionFeatures.feature219RapidPCRTitle, description: AppStrings.NewVersionFeatures.feature219RapidPCRDescription)
```

Comment:
I did not remove the String from AppStrings to not trigger anything in the translation system.